packages/connectivity/src/scp-cf/destination/destination-cache.spec.ts

```diff
@@ -210,11 +205,26 @@ describe('destination cache', () => {
       expect(c2).toBeUndefined();
     });
 
+    it('does not use cache if caching is explicitly disabled', async () => {
+      mockDestinationsWithSameName();
+
+      await getDestination({
+        destinationName: 'SubscriberDest',
+        jwt: subscriberUserToken,
+        useCache: false
+      });
+
+      const c1 = await getSubscriberCache('tenant');
```

Comment:
[req] This test is technically incorrect, because when you set a user jwt and no isolationStrategy, the `isolationStrategy` will be `tenant-user`. So even if something was cached, this line would still return undefined. Either explicitly set the strategy or call getCache methods with tenant-user

Reply:
Also check if you really need this test, since there is another which tests for false behavior. Also please consider the ordering of tests and where you place them. Place similar tests together.

Reply:
Removed the faulty test and added a test for disabling cache in the right group.